views/repo.html

```diff
@@ -67,13 +77,20 @@ <h3 class="panel-title">Deployment Badge</h3>
                 <p>Embed this badge on the project's README or on related tutorials and blog posts.</p>
                 <div class="input-group input-group-sm">
                   <span class="input-group-addon" id="image-url">Image URL:</span>
-                  <input type="text" class="form-control" readonly="readonly" value="{{../protocolAndHost}}/stats/{{url_hash}}/badge.svg" aria-describedby="image-url">
+                  <input type="text" class="form-control" readonly="readonly" value="{{../protocolAndHost}}/stats/{{url_hash}}/badge.svg" aria-describedby="image-url" />
+                  <span class="input-group-addon copy">
+                    <span class="glyphicon glyphicon-copy" data-clipboard-text="{{../protocolAndHost}}/stats/{{url_hash}}/badge.svg"></span>
+                  </span>
+                </div>
                 </div>
                 <br>
                 <div class="input-group input-group-sm">
-                  <span class="input-group-addon" id="markdown">Markdown:</span>
+                  <span class="input-group-addon">Markdown:</span>
                   <input type="text" class="form-control" readonly="readonly" value="![Bluemix Deployments]({{../protocolAndHost}}/stats/{{url_hash}}/badge.svg)
-" aria-describedby="markdown">
+" aria-describedby="markdown" />
```

Comment:
I have always been a fan of closing void tags even though its not required with HTML. 

http://blog.teamtreehouse.com/to-close-or-not-to-close-tags-in-html5